src/usr/bin/gce-nic-naming

```diff
@@ -457,7 +461,14 @@ function generate_name() {
   if [[ ${accelerator_devices[*]} == "" ]] ; then
     if [[ " ${ETHERNET_DEVICES_VENDORS[*]} " =~ \
     [[:space:]]${eth_device_vendor}[[:space:]] ]]; then
-      name_builder="rdma${eth_index}"
+      
+      if [[ "${SUBSYSTEM}" == "net" ]] && [[ ! -d "${SYS_PREPEND_PATH}${DEVPATH}/device/${PCI_TEST_ITEM}" ]]; then
+        # If this is a VF device and not an RDMA the race condition is addressed by naming to position based name.
```

Comment:
Can you explain the race condition, for readers that aren't google internal?

Reply:
Done.